### PR TITLE
create models for FODDEE

### PR DESCRIPTION
**Pull request by adejoke2**

### CODE REVIEW

**Review by usman1607 — COMMENTED** [review]
Add CreatedBy, ModifiedBy, DateCreated and DateLastModified to your BaseEntity.

File: Models/MenuMenuItem.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FOODEE.Models
+{
+    public class MenuMenuItem
```
Comment (usman1607) [inline Models/MenuMenuItem.cs:8]: MenuMenuItem also should inherit from base entity

File: Models/BaseEntity.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FOODEE.Models
+{
+    public class BaseEntity
+    {
```
Comment (usman1607) [inline Models/BaseEntity.cs:9]: Add CreatedBy, ModifiedBy, DateCreated and DateLastModified to your BaseEntity.

File: Models/Payment.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FOODEE.Models
+{
+    public class Payment: BaseEntity
+    {
```
Comment (usman1607) [inline Models/Payment.cs:9]: Why is the entity empty?

**Review by laken11 — APPROVED** [review]

**Pull request merged**